patches/min-vats-online.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/packages/SwingSet/src/kernel/vat-warehouse.js b/packages/SwingSet/src/kernel/vat-warehouse.js
+index 327b74fd4..c6d6668bb 100644
+--- a/packages/SwingSet/src/kernel/vat-warehouse.js
++++ b/packages/SwingSet/src/kernel/vat-warehouse.js
+@@ -243,7 +243,7 @@ export function makeVatWarehouse({
+   panic,
+   warehousePolicy,
+ }) {
+-  const { maxVatsOnline = 50, restartWorkerOnSnapshot = true } =
++  const { maxVatsOnline = 5, restartWorkerOnSnapshot = true } =
```

Comment:
I don't know whether this PR is a good idea, but if it goes forward I would suggest setting the option instead of changing the default.

I.e. adding a `warehousePolicy` to the `runtimeOptions` param here: https://github.com/Agoric/agoric-sdk/blob/50f94936e038d6b8872dfde77b5f54811f339ccd/packages/cosmic-swingset/src/launch-chain.js#L221-L233